Server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "server part of flutter-II module",
+  "scripts": {
+    "start": "node ./bin/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "izetrom",
+  "license": "ISC",
+  "dependencies": {
+    "cookie-parser": "^1.4.5",
+    "express": "^4.17.1",
+    "fs": "^0.0.1-security",
+    "morgan": "^1.10.0",
+    "path": "^0.12.7"
```

Comment:
`path` is already part of node, and it is not even used in the project. Please remove it from the dependencies.
```suggestion
```